apps/cowswap-frontend/src/modules/injectedWidget/utils/validatePartnerFee.test.ts

```diff
@@ -54,6 +55,94 @@ describe('validatePartnerFee()', () => {
     })
   })
 
+  describe('When bps is a map', () => {
```

Comment:
Can you add a test for when both bps and recipient are maps?

Reply:
<img width="571" alt="image" src="https://github.com/cowprotocol/cowswap/assets/7122625/b2596387-17b6-4628-9668-b01bd599e9f5">

There are both of them

Reply:
Thanks, added